hypothetical-use-cases/trade-certificates/examples/steel-mill-cert-test-report/unsigned.vc.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3c-ccg.github.io/vc-json-schemas/context/hypothetical-examples/trade-certificates-v0.0.jsonld"
+  ],
+  "id": "https://w3c-ccg.github.io/vc-json-schemas/credentials/steel-mill-cert-test-report.example.vc.jsonld",
+  "type": ["VerifiableCredential", "CertifiedMillTestReport"],
+  "issuer": "did:key:z6MksUNTUBMbQTzpwAnQ6VFp7aVXTEi7gZbAwpfXkftwDKzK",
+  "issuanceDate": "2019-12-02T15:08:37.751Z",
+  "credentialSchema": {
+    "id": "https://w3c-ccg.github.io/vc-json-schemas/schema/steel-mill-cert-test-report.json",
```

Comment:
and here

Reply:
https://github.com/w3c-ccg/vc-json-schemas/issues/17